src/histolab/util.py

```diff
@@ -38,6 +41,26 @@ def np_to_pil(np_img):
     return Image.fromarray(np_img)
 
 
+def filter_threshold(img: PIL.Image.Image, threshold: float) -> np.ndarray:
```

Comment:
Maybe it's me, but the name `threshold` is odd IMHO. I would expect a number or a range rather than a mask as return value of this util function.

Reply:
A more meaningful name could be appreciate